### ISSUE TITLE
Treat Guardfile* as Ruby code

### ISSUE THREAD

**Issue by mcandre** [atom/language-ruby#84]
language-ruby does a good job loading `Guardfile`s as Ruby code. In large projects, I like to name these `Guardfile-test`, `Guardfile-lint`, `Guardfile-doc`, etc. Could we please treat `Guardfile*` as Ruby code, so these files are colorized as well?

**Comment by kevinsawicki** [comment]
Currently glob patterns aren't supported, see https://github.com/atom/first-mate/pull/25 for more details

**Comment by envygeeks** [comment]
Even just Guardfile at it's basic would be great.

**Comment by mnquintana** [comment]
@envygeeks Are you not getting syntax highlighting for Guardfiles? [It is in the list of recognized filenames.](https://github.com/atom/language-ruby/blob/master/grammars/ruby.cson#L13)

**Comment by envygeeks** [comment]
I wasn't for a while even with that line but it works now so don't worry about me!

**Comment by 50Wliu** [comment]
The new issue that this is blocked on is now atom/first-mate#64.
